desc/protoparse/proto.y

```diff
@@ -873,7 +873,7 @@ rpcOptions : rpcOptions rpcOption {
 	}
 	| rpcOption
 	| {
-		$$ = nil
+		$$ = []*optionNode{}
```

Comment:
I think you technically need the same thing below, on line 883.

Reply:
Done